torchgeo/samplers/single.py

```diff
@@ -89,10 +93,12 @@ def __init__(
             length: number of random samples to draw per epoch
             roi: region of interest to sample from (minx, maxx, miny, maxy, mint, maxt)
                 (defaults to the bounds of ``dataset.index``)
+            units: defines if `size` is in pixels or in CRS units.
```

Comment:
List the potential options here

Reply:
Docs should now add type hints to this, so the options should be more clear.